relay-event-normalization/src/normalize/span/description/sql/mod.rs

```diff
@@ -698,6 +698,46 @@ mod tests {
         "UPDATE tbl SET foo = CASE WHEN .. THEN .. END"
     );
 
+    scrub_sql_test!(
+        duplicate_conditions,
+        r#"SELECT *
+        FROM a
+        WHERE a.status = %s
+        AND (
+            (a.id = %s AND a.org = %s)
+            OR (a.id = %s AND a.org = %s)
+            OR (a.id = %s AND a.org = %s)
+            OR (a.id = %s AND a.org = %s)
+        )"#,
+        "SELECT * FROM a WHERE status = %s AND ((id = %s AND org = %s))"
```

Comment:
are the double parentheses `((` left on purpose ?

Reply:
I did not intend to leave them initially, but they might help indicate to the user that something was scrubbed here. They should be easy to remove though, if we want to @gggritso any opinion?

Reply:
I'd personally remove them. I think it's easy to notice queries are getting scrubbed so it's no surprise.

Reply:
Done.

Reply:
No strong opinion here 👍🏻